samples/python/text2image/README.md

```diff
@@ -28,7 +32,7 @@ optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task sta
 
 ## Run
 
-`python main.py ./dreamlike_anime_1_0_ov/FP16 "cyberpunk cityscape like Tokyo New York with tall buildings at dusk golden hour cinematic lighting"`
```

Comment:
why `python` is not required? I see other READMEs for Python sample also mention it

Reply:
It works without '`python`' on windows, but looks like on linux '`python`' is required in this command, just tried to run it and it failed.

Pastushenkov Dmitriy asked to align it, as in some samples we have '`python`' in other samples don't
https://github.com/openvinotoolkit/openvino.genai/blob/1729e78409c514d2f02d833ede179e4d87631c02/samples/python/multinomial_causal_lm/README.md?plain=1#L20

So I guess we should correct other samples and add '`python`' there.

Reply:
@Wovchena are ok to add python everywhere?

Reply:
You must run `chmod +x` and push the change to enable the feature on Linux.

Linux builds must test samples without `python` to verify shebang lines and execution permissions. Windows GHA can't do that: https://github.com/actions/setup-python/issues/900.

It's controversial for README. Some Linux distributives still have `python3` instead of `python` so providing a single command isn't fully possible. Omitting `python` isn't applicable for every Windows set up. I guess that's up to you, just make it consistent.

Reply:
> providing a single command isn't fully possible

but we demonstrate running within env, and I suppose each env (Windows, Linux, macOS) has `python`, does not it?

Reply:
You could say that env creation goes without saying :) In that case having `python` is preferred.